proof/refine/RISCV64/RAB_FN.thy

```diff
@@ -78,7 +78,7 @@ declare resolveAddressBitsFn.simps[simp del]
 
 lemma isCNodeCap_capUntypedPtr_capCNodePtr:
   "isCNodeCap c \<Longrightarrow> capUntypedPtr c = capCNodePtr c"
-  by (clarsimp simp: isCap_simps)
+  by (clarsimp simp: isCap_simps State_H.RISCV64_H.PPtr_def)
```

Comment:
```suggestion
  by (clarsimp simp: isCap_simps RISCV64_H.PPtr_def)
```

Do you remember if the `RISCV64_H` is really necessary here?

Reply:
Turns out the `RISCV64_H` is necessary, but I've removed the `State_H`